docs/source/releases/v1_10_0_dev0.rst

```diff
@@ -71,15 +70,9 @@ Add a schema for validating `Gridlines` plugins.
 
 *From issue NRLMMD-GEOIPS/geoips#109: 2023-04-04, Add a schema for Gridlines plugins.*
 
-Add a validator for YAML-based plugins.
```

Comment:
This was duplicated, the other one is still there:

https://github.com/NRLMMD-GEOIPS/geoips/blob/122-convert-yaml-files-for-product_params-interface/docs/source/releases/v1_10_0_dev0.rst#add-a-validator-for-yaml-based-plugins